src/components/DonationForm.jsx

```diff
@@ -0,0 +1,122 @@
+/* eslint-disable react/jsx-props-no-spreading */
+import React from 'react';
+import {
+  FormLabel,
+  Input,
+  InputGroup,
+  Select,
+  FormControl,
+  FormErrorMessage,
+  Button,
+} from '@chakra-ui/react';
+import { yupResolver } from '@hookform/resolvers/yup';
+import { useForm } from 'react-hook-form';
+import * as yup from 'yup';
+import './DonationForm.css';
```

Comment:
Use module.css, see the [PNP Best Programming Practices](https://drive.google.com/file/d/1EfQjal0X2wpreBFlGLLqZznisbncYdFf/view) for more info